root/io/evolution/rules/execSourceTypes.cxx

```diff
@@ -454,6 +514,9 @@ int readfile(const char *filename = "sourcetypes.root")
    res = res && check(n->fListExplicit, {"b1", "b2", "b3"});
    res = res && check(n->fListImplicitExplicit, {"c1", "c2", "c3"});
 
+   res = res && check_array<3>(n->fHitPattern, std::vector<int>{111, 112, 113});
```

Comment:
Possibly use initializer list or `std::array<int, 3>` instead.
```suggestion
   res = res && check_array<3>(n->fHitPattern, std::initializer_list<int>{111, 112, 113});
```

Reply:
`std::initializer_list` does not worked because the type of the 2nd parameter is a template parameter of the function.  `std::array<int, 3>` could work but have the annoying need to repeat the `3` (actually I would love to not have it in the `check_array<3>` either but the naive tentatives to remove it did not work out.